File: cpp/cicle/Whl.cpp
#include "Whl.hpp"
#include <iostream>

using namespace std;

void Whl::main(int argc, char **argv) {
    char* s;
    do {
        cout << "Hell World!" << endl;
        cout << "Doriti sa printati?";
        cin >> s;
    }
    while (strcmp(s, "y") == 0);

}

File: src/main/java/cicle/Whl.java
package cicle;

import java.util.Scanner;

public class Whl {
    static Scanner scanner = new Scanner(System.in);
    public static void main(String[] args) {
        String s;
        do{
            System.out.println("Hello World!");
            System.out.println("Doriti sa printati?");
            s = scanner.nextLine();
        }while (s.equals("y"));
        System.out.println("Is end!");
    }
}
